apph-back/src/main/java/com/viseo/apph/dao/PhotoDao.java

```diff
@@ -18,9 +19,9 @@ public Photo addPhoto(Photo photo) {
         return photo;
     }
 
-    public List<Photo> getUserPhotos(long idUser) throws NoResultException {
-        return em.createQuery("SELECT p FROM Photo p WHERE p.idUser=:idUser", Photo.class)
-                .setParameter("idUser", idUser)
+    public List<Photo> getUserPhotos(User user) throws NoResultException {
```

Comment:
```suggestion
    public List<Photo> getUserPhotos(User user) {
```
`getResultList()` ne renvoie jamais de `NoResultException`, il renvoi une liste vide s'il trouve rien